x-pack/test/api_integration/apis/telemetry/telemetry_local.js

```diff
@@ -79,9 +79,7 @@ export default function({ getService }) {
       expect(stats.stack_stats.kibana.plugins.apm.services_per_agent).to.be.an('object');
       expect(stats.stack_stats.kibana.plugins.infraops.last_24_hours).to.be.an('object');
       expect(stats.stack_stats.kibana.plugins.kql.defaultQueryLanguage).to.be.a('string');
-      expect(stats.stack_stats.kibana.plugins['maps-telemetry'].attributes.timeCaptured).to.be.a(
-        'string'
-      );
+      expect(stats.stack_stats.kibana.plugins['maps-telemetry'].timeCaptured).to.be.a('string');
```

Comment:
do you mind adding a test-case here that ensures the `attributes` object does not exist

e.g.:
`expect(stats.stack_stats.kibana.plugins['maps-telemetry'].attributes).to.be(undefined);`

and that the id and type fields are stripped.

```
expect(stats.stack_stats.kibana.plugins['maps-telemetry'].id).to.be(undefined);
expect(stats.stack_stats.kibana.plugins['maps-telemetry'].type).to.be(undefined);
```

Reply:
I wonder if we can rename `'maps-telemetry'` to `'maps'`. It's a bit redundant :)
We can easily apply the same rename pattern at the same time we are removing the `.attributes` bit in the existing documents

Reply:
> do you mind adding a test-case here that ensures the attributes object does not exist

Done!

> and that the id and type fields are stripped

Done!

> I wonder if we can rename 'maps-telemetry' to 'maps'. It's a bit redundant :)

Agreed and done!